app/src/main/res/drawable/ic_bell.xml

```diff
@@ -0,0 +1,18 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="16dp"
+    android:height="20dp"
```

Comment:
icons should ideally have `width="24dp"`  `height="24dp"`  which is recommended by [material 3](https://m3.material.io/styles/icons/designing-icons)